### PR TITLE
"About MuseScore" dialog strings adjustments to meet MStore requirements

### PR DESCRIPTION
**Pull request by anatoly-os**
Merge strings back. The ones with the links to .org are removed from Microsoft Store package

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
To truely remove them from builds for Windows Store, whouldn't `#if !defined(FOR_WINSTORE)` be needed?

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -4489,12 +4489,14 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
-                              .arg(tr("Visit %1www.musescore.org%2 for new versions and more information.\n").arg("<a href=\"http://www.musescore.org/\">")
+                              .arg(tr("Visit %1www.musescore.org%2 for new versions and more information.\n"
+                                      "Support MuseScore with your %3donation%4.\n\n")
+                                   .arg("<a href=\"http://www.musescore.org/\">")
+                                   .arg("</a>")
+                                   .arg("<a href=\"http://www.musescore.org/donate\">")
```
[the remark this answers is not in the document]
Comment (anatoly-os) [reply mscore/musescore.cpp]: They decline all builds which have any mentions of the Musescore.org website, because the big "Download" button is there.

**Comment by Jojo-Schmitz** [comment]
And better leaveout as many of the linefeeds from the to-be-trandlated text as possible

File: mscore/musescore.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply mscore/musescore.cpp]: How stupid is that :-(

**Comment by thomasbonte** [comment]
Could you turn all http to https?

**Comment by Jojo-Schmitz** [comment]
And better leaveout as many of the **trailing** linefeeds from the to-be-translated text as possible

**Comment by anatoly-os** [comment]
- Replaced `http` with `https`
- Placed "suspicious" strings under `#if !defined(FOR_WINSTORE)`
- Reorganised strings to exclude two trailing linefeeds from translating string

File: mscore/musescore.cpp [hunk 2]
```diff
@@ -4488,13 +4488,20 @@ AboutBoxDialog::AboutBoxDialog()
...
+      QString copyrightString = tr("Copyright &copy; 1999-2019 Werner Schweer and Others.\n"
+                                   "Published under the GNU General Public License.").replace("\n", "<br/>");
       copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
-                              .arg(tr("Visit %1www.musescore.org%2 for new versions and more information.\n").arg("<a href=\"http://www.musescore.org/\">")
-                                   .arg("</a>").replace("\n", "<br/>") +
-                                   tr("Support MuseScore with your %3donation%4.\n\n").arg("<a href=\"http://www.musescore.org/donate\">")
-                                   .arg("</a>").replace("\n", "<br/>") +
-                                   tr("Copyright &copy; 1999-2019 Werner Schweer and Others.\n").replace("\n", "<br/>") +
-                                   tr("Published under the GNU General Public License.").replace("\n", "<br/>")));
+                              .arg(visitAndDonateString + copyrightString));
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: Does this work with `visitAndDonateString` being potentially empty and not initialized?

Comment (anatoly-os) [reply mscore/musescore.cpp]: Sure, `QString` is just empty if initialised with default constructor.

Comment (anatoly-os) [reply mscore/musescore.cpp]: Done

**Pull request merged**